warehouse-management-service/main.go

```diff
@@ -4,45 +4,57 @@ import (
 	"fmt"
 	"net/http"
 	"os"
+	"warehouse-management-service/internal/config"
+	"warehouse-management-service/internal/handler"
+	"warehouse-management-service/pkg/database/postgres"
 
-	"github.com/gin-gonic/gin"
+	_ "github.com/lib/pq"
+)
+
+const (
+	CONFIG_FILE_PATH_ENV_VARIABLE = "CONFIG_FILE_PATH"
```

Comment:
style nit: In Go, it's idiomatic to have CapitalCase constant names. Check out some stdlib code for examples of how they could be named.

We should follow the convention and change this to `ConfigFilePathEnvVariable`, or better yet, with some prefixing convention like `EnvConfigFilePath`.

Reply:
Had mixed up the Go constant naming convention with a different one. Renamed it